ee/tabby-webserver/src/service/background_job/mod.rs

```diff
@@ -48,6 +49,26 @@ pub enum BackgroundJobEvent {
     Daily,
 }
 
+impl Display for BackgroundJobEvent {
```

Comment:
why we need this?

Reply:
because we use this in the notification content to show users which job has failed.

job name only has the type, can not be used to distinguish two job with the same type;
job command is in json format

Reply:
ok, do not use Display interface for the purpose, instead, just pass in the job name in `match` clause of `BackgroundJobEvent`

Reply:
And please do not include job id / integration id in job name.

Simply saying something like `Git repository synchronize failed" would be sufficient.